File: UnicoVehicle/UnicoVehicle.DTO/Showroom.cs
using System;
namespace UnicoVehicle.DTO
{
    public class Showroom
    {
        public int ShowroomId { get; set; }
        public Miscellaneous.Company Company { get; set; }
        public Miscellaneous.District District { get; set; }
        public string ShowroomName { get; set; }
        public string Address { get; set; }
        public int PINCODE { get; set; }
        public string Manager { get; set; }
        public bool hasSales { get; set; }
        public bool hasService { get; set; }
        public bool isOperational { get; set; }
    }
}

File: Flutter App/unicoVehicle/lib/Models/Showroom.dart
import './Miscellaneous/Company.dart';
import './Miscellaneous/District.dart';

class Showroom {
  final int? showroomId;
  final Company? company;
  final District? district;
  final String? showroomName;
  final String? address;
  final String? manager;
  final int? pincode;
  final bool? hasSales;
  final bool? hasService;
  final bool? isOperational;

  Showroom({
    this.address,
    this.company,
    this.district,
    this.hasSales,
    this.hasService,
    this.isOperational,
    this.manager,
    this.pincode,
    this.showroomId,
    this.showroomName,
  });
}
